x-pack/plugin/autoscaling/build.gradle

```diff
@@ -19,7 +19,6 @@ dependencies {
   testImplementation project(path: xpackModule('ilm'))
 
   testImplementation "com.fasterxml.jackson.core:jackson-databind:${versions.jackson}"
-  testImplementation project(':plugins:transport-nio') // for http
```

Comment:
Am I right in thinking that this is a left-over from #82088? Or was there never a need for this?

Reply:
I don't think this was ever needed. The one test actually using HTTP `org.elasticsearch.xpack.autoscaling.action.GetAutoscalingCapacityRestCancellationIT` was always using Netty as far as I can tell from the git history. The same is true for other x-pack plugin `build.gradle` adjusted here.